agent/agents/process/process_test.go

```diff
@@ -189,3 +189,46 @@ func TestProcess(t *testing.T) {
 		require.EqualError(t, err, "os: process already finished", "child process with pid %v is not killed", pid)
 	})
 }
+
+func TestMatchLogLevel(t *testing.T) {
```

Comment:
can you please refactor that to a table test?

Reply:
done